integration/spark/src/main/scala/org/apache/spark/sql/CarbonEnv.scala

```diff
@@ -95,6 +95,12 @@ class CarbonEnv {
     sparkSession.udf.register("text_match", new TextMatchUDF)
     sparkSession.udf.register("text_match_with_limit", new TextMatchMaxDocUDF)
     sparkSession.udf.register("in_polygon", new InPolygonUDF)
+    sparkSession.udf.register("in_polygon_list", new InPolygonListUDF)
+    sparkSession.udf.register("in_polyline_list", new InPolylineListUDF)
+    sparkSession.udf.register("in_polygon_range_list", new InPolygonRangeListUDF)
+
```

Comment:
register code should not at here, here is the overall process should not contain details func

Reply:
More and more, consider to use a function processing separately

Reply:
have used a separate function to process